ics-frontend/src/app/image-classification/image-classification.component.ts

```diff
@@ -37,6 +36,15 @@ export class ImageClassificationComponent {
     );}   
   }
 
+  navigateToImageClassification(imageId: number) {
```

Comment:
Is this used somewhere?

Reply:
this is the first variant of the navigation I had implemented, not used anymore